src/app/lib/utilities/passport/index.test.js

```diff
@@ -54,4 +54,8 @@ describe('isValidPassportHome', () => {
   it('should give true for null passportHome', () => {
     expect(isValidPassportHome(null, 'portuguese')).toEqual(true);
   });
+
+  it('should give true for null service', () => {
+    expect(isValidPassportHome('brasil', null, ['brasil'])).toEqual(true);
```

Comment:
We need a case for `isValidPassportHome(null, 'portuguese', ['brasil'])` (which is the test case that broke Portuguese)

Reply:
Fixed!